### PR TITLE
feat: remove lndhub integration before service sunsets in april

### PR DESCRIPTION
**Pull request by 0xZaddyy**
### Describe the changes you have made in this PR

The LNDHUB service is scheduled to sunset in April. As a result, this commit removes the LNDHUB integration from the payment processing module to ensure continued functionality of the app. This change is necessary to avoid any disruptions to the app's performance.

### Link this PR to an issue [optional]

Fixes Issue #2194

### Type of change

(Remove other not matching type)

- `feat!`: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Screenshots of the changes [optional]

_Add screenshots to make your changes easier to understand. You can also add a video here._

### How has this been tested?

i made multiple tests after the removal of lndhub and bluewallet and everything worked perfectly

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -3,75 +3,75 @@ description: Create a report to help us improve (use this to report bugs only).
...
+      description: A clear and concise description of what you expected to happen
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Alby information
+      placeholder: |
+        - Alby Version: [e.g. 1.5.0]
+        - Alby installed through: [e.g. installed through the browser stores, installed manually]
+        - Wallet connected with Alby: [e.g. LND
```
Comment (reneaaron) [inline .github/ISSUE_TEMPLATE/bug_report.yml:52]:
```suggestion
        - Wallet connected with Alby: [e.g. LND, Core-Lightning, Lightning Terminal, LNBits, etc.]
```

File: src/app/router/connectorRoutes.tsx [hunk 2]
```diff
@@ -61,25 +58,15 @@ function getConnectorRoutes() {
...
       element: <ConnectKollider />,
       title: i18n.t("translation:choose_connector.kollider.title"),
       description: i18n.t("translation:choose_connector.kollider.description"),
       logo: kolliderLogo,
     },
-    {
-      path: "lnd-hub-bluewallet",
-      element: <ConnectLndHub />,
-      title: i18n.t("translation:choose_connector.lndhub_bluewallet.title"),
-      logo: lndhubBlueWallet,
```
Comment (reneaaron) [inline src/app/router/connectorRoutes.tsx:81]: Can we remove the bluewallet logo as well or is that used somewhere else?

File: src/app/router/connectorRoutes.tsx [hunk 3]
```diff
@@ -61,25 +58,15 @@ function getConnectorRoutes() {
...
       path: "kollider",
       element: <ConnectKollider />,
       title: i18n.t("translation:choose_connector.kollider.title"),
       description: i18n.t("translation:choose_connector.kollider.description"),
       logo: kolliderLogo,
     },
-    {
-      path: "lnd-hub-bluewallet",
-      element: <ConnectLndHub />,
-      title: i18n.t("translation:choose_connector.lndhub_bluewallet.title"),
```
Comment (reneaaron) [inline src/app/router/connectorRoutes.tsx:80]: Can we remove the translations as well then? (see `translation.json`s)

**Review by reneaaron — CHANGES_REQUESTED** [review]
Thanks for the PR @0xZaddyy! 🙌 Just did a first review and left some comments. 

I'd probably wait for the sunset to merge this PR as users could use Alby to move the money out of their LNDHub accounts at BlueWallet.

File: src/app/router/connectorRoutes.tsx (hunk 3 above)
Comment (0xZaddyy) [reply src/app/router/connectorRoutes.tsx:80]: alright

File: src/app/router/connectorRoutes.tsx (hunk 2 above)
Comment (0xZaddyy) [reply src/app/router/connectorRoutes.tsx:81]: okay sir, i'll remove it

Comment (0xZaddyy) [reply src/app/router/connectorRoutes.tsx:81]: all changes has been made, thanks

**Review by bumi — COMMENTED** [review]
can we fully get rid of the `lndHubType` ?

**Comment by reneaaron** [comment]
Sunset has been moved to end of May: 

https://twitter.com/bluewalletio/status/1652969929287630848

**Comment by bumi** [comment]
I think it's now time to get this in and delete the BlueWallet logo from the screens. 
The lndhub connector will still be needed.
@0xZaddyy do you mind rebasing this one?

**Comment by reneaaron** [comment]
Closing due to inactivity in favor of #2875.

**Pull request closed without merging**